common/config/kafkaConfig.go

```diff
@@ -99,3 +102,8 @@ func (k *KafkaConfig) GetBrokersForKafkaCluster(kafkaCluster string) []string {
 func (k *KafkaConfig) GetTopicsForApplication(app string) TopicList {
 	return k.Applications[app]
 }
+
+// GetKafkaClusterSecureConfigForTopic gets isSecure status from topic
+func (k *KafkaConfig) GetKafkaSecureConfigForTopic(topic string) bool {
+	return k.Topics[topic].IsSecure
```

Comment:
huh.  is this a general kafka-topic flag, or is it something entirely internal?  may be worth making a more general config-map or something if it's internal.

Reply:
It is entirely internal change. New topics are enforced to be secure. Config maps means converts it into a map[string]string?

Reply:
+1 for not exposing such internal only fields to avoid bloating the public contract. 
`map[string]any Properties` is a good idea

Reply:
Updated it to use map in https://github.com/uber/cadence/pull/5474